plugins/modules/postgresql_ping.py

```diff
@@ -83,6 +83,11 @@
   returned: always
   type: dict
   sample: { major: 13, minor: 2, full: '13.2', raw: 'PostgreSQL 13.2 on x86_64-pc-linux-gnu' }
+conn_err_msg:
+  description: Connection error message.
+  returned: always
+  type: str
+  sample: ''
```

Comment:
```suggestion
  sample: ''
  version_added: 1.7.0
```

Reply:
Ah, forgot that it's needed for this section too:) Thanks!